credentials/src/test/java/me/uport/sdk/credentials/CredentialsTest.kt

```diff
@@ -566,4 +566,124 @@ class CredentialsTest {
             isInstanceOf(JWTAuthenticationException::class)
         }
     }
+
+    @Test
+    fun `claims request params builds request successfully`() = runBlocking {
+        val claim = ClaimsRequestParams()
+            .addVerifiable(
+                "email",
+                VerifiableParams(
+                    "We need to be able to email you",
+                    true
+                )
+                    .addIssuer("did:web:uport.claims", "https://uport.claims/email")
+                    .addIssuer("did:web:sobol.io", "https://sobol.io/verify")
+            )
+            .addVerifiable(
+                "nationalIdentity",
+                VerifiableParams(
+                    "To legally be able to open your account"
+                )
+                    .addIssuer("did:web:idverifier.claims", "https://idverifier.example")
+            )
+            .addUserInfo("name", UserInfoParams("Show your name to other users", true))
```

Comment:
since UserInfoParams can only be used in this context, it makes sense to have the parameters be directly passed to the `.addUserInfo()` method.
This call would then look like this:
```suggestion
            .addUserInfo("name", "Show your name to other users", true)
```

It is a bit more concise and can provide the same parameter type safety